src/views/MainPage/MainPage.js

```diff
@@ -25,36 +35,19 @@ function renderMainView() {
     <div class="info">
 
       <div class="bubbles">
-        <div class="bubble fact">
-          <div class="bubble-img">
-            <img src="${FACT_ANIMAL_URL}" alt="the animal the sentence is about">
-          </div>
-          <div class="bubble-text">
-            <h2>Did you know?</h2>
-          </div>
-        </div>
-
-        <div class="bubble adoption">
-          <div class="bubble-img">
-            <img src="${ADOPTION_ANIMAL_URL}" alt="the animal to adoption">
-          </div>
-          <div class="bubble-text">
-            <h2>Or maybe you'd like to adopt your own pet?</h2>
-            <p>This one is looking for home</p>
-            <p>See more here: <a href="#">Adoption Page</a></p>
-          </div>
-        </div>
       </div>
 
       <div class="animal">
-        <img src="${MAIN_ANIMAL_PATH}" alt="kangoroo that tells informations" /></div>
       </div>        
 
     </div>
   
   </div>
   `;
+}
 
+// Render Navigation
```

Comment:
well, everyone can expect that renderNavigation does Render Navigation so this comment is needless